Umbraco-StarterKit/Manage-users/index.md

```diff
@@ -4,4 +4,25 @@ versionFrom: 8.0.0
 
 # Manage users
 
-In this section you can find all the information and details you need in order to work with and manage both users and members on your website.
+In this section, you can find all the information and details you need in order to work with and manage both users and members on your website.
+
+## What is a User
+
+A user is someone who has access to the Umbraco backoffice and is not to be confused with Members. When you enter your Umbraco project you will find that a user allready excist made with the login (email) and password entered during creation of the project. Users can be created, edited, and managed in the User section.
+Managing your users.
+You can manage your users so they can’t publish or edit only read, or you can disable someone from your project if they should not have access anymore.
+
+Please note that for now a user cannot be deleted, they can only be disabled
+The reason for this is that, if the user were deleted then everything they created would be deleted along with the user.
+
+Please see the [User](../Manage-users/Users/index.md) section for more information about users and what you can do with them.
+
+:::note You will find that your project has a user named themeowner. It is there because this is what implements the entire theme on your project. If you remove this user your project won't have access to the theme anymore and that will make your project cease to function:::
```

Comment:
```suggestion
:::note 
You will find that your project has a user named themeowner. It is there because this is what implements the entire theme on your project. 
:::
```